src/main/java/co/omise/models/PaymentMethod.java

```diff
@@ -0,0 +1,56 @@
+package co.omise.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Represents Omise Payment method object that is found in the response for the Capability API
+ *
+ * @see <a href="https://www.omise.co/capability-api">Capability API</a>
+ */
+public class PaymentMethod extends Model {
```

Comment:
I'm not sure if we should have a separate type for Card and Installments methods here

Reply:
We can do that, but I fear it might kind of limit us if more method become available.